breathless/index.md

```diff
@@ -0,0 +1,87 @@
+
+A DX avisará do risco da acción. Logo escollerás unha habilidade e rolarás o dado que encaixe co seu valor. Se unha aliada axuda, tamén fai unha tirada e comparte o risco contigo. Tomade o dado máis alto e interpretade os resultados:
+
+* Nun 1 ou 2 fracasades e algo malo sucede.
+* Nun 3 ou 4 tedes éxito, pero cun custe.
+* Nun 5 ou máis tedes éxito. Canto máis alto sexa o número, mellor o efecto.
+
+Sobrevivir é difícil. Todo o mundo que rolou diminúe o valor da habilidade que empregaron nun nivel: d12 » d10 » d8 » d6 » d4. As habilidades non poden baixar dun d4.
+
+A DX pode deixar unha decisión nas mans da sorte. Para facelo, elixe un dado baseándose nas posibilidades de que suceda algo, rola e interpreta os resultados.
+
+
+## Coller folgos
+Para devolver todas as túas habilidades ó seu valor orixinal, podes pedir un tempo para *coller folgos*. Esto será unha pequena pausa nunha tensión. Pode ser feito en calquera momento, incluso en combate.
+
+Cando unha personaxe colla folgos, a DX analiza a escena e introduce unha nova complicación para o grupo.
+
+
+## Roladas de botín
+Cando a ficción o permita, podes facer unha rolada de botín para atopar ferramentas, armas, kits de primeiros auxilios, etc. Pero ten coidado co que se agocha nas sombras.
+
+Cando fagas unha rolada de equipamento, lanza un d20.
+* Nun 1 algo malo sucede.
+* Nun 16 ou 17 atopas un obxecto d6.
+* Nun 18 ou 19 atopas un obxecto d8.
+* Nun 19 ou 20 atopas un obxecto d10 ou un kit de primeiros auxilios.
+
+
+## Mochila
+Os obxectos na túa mochila poden ser empregados no lugar das túas habilidades. Comezan cun valor de dado e diminúen cada vez que os empregues. Cando un obxecto chegue a d4 rómpese e non pode volver ser usado.
+
+As personaxes só poden levar 3 obxectos e un kit de primeiros auxilios ó mesmo tempo na súa mochila.
+
+
+## Acrobacias
+Cando fagas unha acrobacia rola un d12 no lugar do que indique a habilidade correspondente. Unha vez feita, precisarás *coller folgos* antes de facer outra.
+
+
+## Estrés
+Cando te enfrontes a unha consecuencia derivada dunha das túas accións, a DX pode decidir que tomes 1 punto de estrés. Se a túa personaxe chega a 4 puntos de estrés tórnase *vulnerable*.
+
+Cando esteas *vulnerable*, fracasar nunha acción perigosa pode significar que quedas fóra de combate ou morto.
+
+Podes empregar un kit de primeiros auxilios para baixar 2 puntos de estrés, ou permanecer nun lugar seguro un certo tempo para baixar 1 punto de estrés.
+
+
+## Personaxes
+Na túa folla de personaxe, escribe o seu nome e os seus morfemas.
+
+De inicio todas as habilidades teñen un valor d4. Asigna un d10, un d8 e un d6 a tres habilidades nas que penses que a túa personaxe é boa.
+
+As personaxes teñen un total de 6 habilidades:
+* Golpear: romper, empurrar, forzar.
+* Bulir: correr, saltar, escalar.
+* Escapulir: espiar, agocharse, disfrazarse.
```

Comment:
«Escapulirse» parece ser un verbo apenas pronominal, polo que recollen os dicionarios. Estás a propoñer un neoloxismo intransitivo que engadir á xerga roleira (cousa aceptábel) ou foi un descoido (e mellor poñer «escapulirse»)? É só por saber.

Reply:
Lembro ter buscado por algún verbo sen pronome porque non me parecía elegante ter aí un "escapulirse" ou que todos os da lista tamén foran pronominais. Teimas miñas, supoño.

Agora quero cambialo pero non sei ben cal pode ser a alternativa. Quizais "espiar" no seu sentido máis amplo, pero entón gustaríame ter outro na listaxe que non fora pronominal... Algunha idea?

Reply:
Que tal usar «Espiar» e como listaxe «axexar, agocharse, disfrazarse»?

Reply:
Perfecto, grazas!